src/lib/tree/Constants.sol

```diff
@@ -15,7 +15,13 @@ library Constants {
     bytes1 internal constant LEAF_PREFIX = 0x00;
     bytes1 internal constant NODE_PREFIX = 0x01;
 
-    /// @dev Parity share namespace ID
-    NamespaceID internal constant PARITY_SHARE_NAMESPACE_ID =
-        NamespaceID.wrap(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF);
+    /// @dev Parity share namespace in bytes.
+    bytes29 internal constant PARITY_SHARE_NAMESPACE_BYTES =
+        0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF;
+}
+
+/// @dev Parity share namespace.
+/// utility function to provide the parity share namespace as a Namespace struct.
+function PARITY_SHARE_NAMESPACE() pure returns (Namespace memory) {
```

Comment:
Instead of a function, is it possible to turn this into a constant? That way it doesn't have to run every time.

Reply:
We can't have a constant struct. constants are only for native types. That's why I created that function.